docs/advanced/peering/inter-cluster-authentication.md

```diff
@@ -1 +1,231 @@
+:caption: "Cluster consumer"
+liqoctl unauthenticate --kubeconfig $CONSUMER_KUBECONFIG_PATH --remote-context $PROVIDER_KUBECONFIG_PATH
+```
+
+When successful, **the identity** used to operate on the cluster provider, **and the tenant resource** on the provider **are removed**. Therefore, from this point on, the cluster consumer is not authorized to offload and reflect resources on the provider.
+
+## Manual authentication
+
+```{warning}
+The aim of the following section is to give an idea of how Liqo works under the hood and to provide the instruments to address the more complex cases. For the major part of the cases the `liqoctl peer` is enough, so [stick with it](/usage/peer) if you do not have any specific needs.
+```
+
+When the user has not access to both the clusters, it is possible to perform the authentication by calling some commands in each of the clusters.
+
+The authentication process consists of the following steps (Where we call *consumer* the cluster offloding the resource on the *cluster provider*):
+
+1. **Cluster provider**: generates a nonce to be signed
+2. **Cluster consumer**: generates a `Tenant resource` to be applied on the provider cluster. It contains:
+    - the provider's signed nonce (Each cluster in Liqo has a pair of key and certificate, at the moment the certificate is self-signed and it is created at installation time)
+    - a *CSR* for the certificate to use to authenticate to the provider's K8S API server
+3. **Cluster provider**: applies the consumer-generated Tenant resource and if the signature of the nonce is valid, the consumer-provided CSR is signed. And it is possible to generate an Identity resource containing the signed certificate.
+4. **Cluster consumer**: applies the provider-generated Idenitity and the `kubeconfig` to interact with provider K8S API server is created.
+
+### Nonce and tenant namespace generation (cluster provider)
+
+In this step, starting from the cluster provider, we need to:
+
+- Create a **tenant namespace**, which will be used for the resource related to new peering;
+- Create a secret to contain the nonce
+
+Those operation can be performed via `liqoctl`, executing the following command **from the cluster provider**:
+
+```{code-block} bash
+:caption: "Cluster provider"
+liqoctl create nonce --remote-cluster-id $CLUSTER_CONSUMER_ID
+```
+
+When successful the output of the command will be something like the following:
+
+```text
+ INFO  Nonce created
+ INFO  Nonce generated successfully
+ INFO  Nonce retrieved: 5IDZQiNG7EUv+GPmS5RvzcTAo8ksSfM3LUTFaBYnPfvY5WndhNGkJVFOFmTVc0ArT4B/FkH6QO7Kfhk6Q1B4ww==
+```
+
+```{admonition} Tip
+**Take note of the nonce as it can will be used in the following step**.
+```
+
+As result of this command you should be able to see a new **tenant namespace** with name `liqo-tenant-${COMSUMER_ID}` and, inside it, a secret containing the nonce.
+
+```{code-block} bash
+:caption: "Cluster provider"
+$ kubectl get secret -n liqo-tenant-consumer-id
```

Comment:
The name of the namespace can be misleading. I would explicitely put the `${}` or make the user save the namespace in a variable